ci/checks/style.sh

```diff
@@ -12,69 +12,12 @@ PATH=/opt/conda/bin:$PATH
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
 
-# Run flake8 and get results/return code
-FLAKE=`flake8 --exclude=cpp,thirdparty,__init__.py,versioneer.py && flake8 --config=python/.flake8.cython`
-RETVAL=$?
+FORMAT_FILE_URL=https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-22.12/cmake-format-rapids-cmake.json
+export RAPIDS_CMAKE_FORMAT_FILE=/tmp/rapids_cmake_ci/cmake-formats-rapids-cmake.json
+mkdir -p $(dirname ${RAPIDS_CMAKE_FORMAT_FILE})
+wget -O ${RAPIDS_CMAKE_FORMAT_FILE} ${FORMAT_FILE_URL}
 
-# Output results if failure otherwise show pass
-if [ "$FLAKE" != "" ]; then
-  echo -e "\n\n>>>> FAILED: flake8 style check; begin output\n\n"
```

Comment:
I've found black to work well with flake8 - there is some guidance from black on how to best configure this here https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8 to avoid incompatibilities between black and flake8.  While it doesn't seem to be a problem right now,  maybe we should pre-emptively ignore `E203` to the flake8 config here ? 

Black doesn't support cython currently https://github.com/psf/black/issues/359 - and is only configured to run on pure python code here

Reply:
I've definitely run into E203 issues in cudf so I would recommend disabling it.

In general, with the appropriate black configuration I have not run into conflicts with flake8. We've been running both in concert for a long time on cudf now with no issues.

There was a team at Bloomberg that did some initial work to get black working for Cython, but it was never completed and ran into very snags around the minimal parser grammar that a linter would need to support in order to support Python 2, Python 3, and Cython last I checked.

Reply:
I've updated to use the same flake8 config as cudf in the last commit - which includes disabling E203, as well as it being more specific on disabling flake8 warnings for cython.

Reply:
I'd say the current flake8 configuration for cudf (which you copied) is highly battle tested for Cython compatibility. I pulled that together by analyzing a number of Cython repos.